x/lockup/keeper/lock_test.go

```diff
@@ -312,3 +312,50 @@ func (suite *KeeperTestSuite) TestLockTokensAlot() {
 	}
 	// panic(1)
 }
+
+func (suite *KeeperTestSuite) TestEndblockerWithdrawAllMaturedLockups() {
```

Comment:
Can we have a test that shows if that locks with the following unbonding times are unbonded in sequence:

* 5 second lockup
* 1 second lockup
* 3 second lockup

and then test the following two situations:

After 1 second, only the 1 second is freed, 3 seconds the 3 second is freed, 5 seconds the 5 second is freed

After 5 seconds, all three are freed

Reply:
I'm working on adding this rn

Reply:
Added in #355